src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -1335,6 +1335,11 @@ public ErrorDiagnostic UnknownModuleReferenceScheme(string badScheme, ImmutableA
                 "BCP225",
                 $"The discriminator property \"{propertyName}\" value cannot be determined at compilation time. Type checking for this object is disabled.");
 
+            public ErrorDiagnostic MissingDiagnosticCodes() => new(
+                TextSpan,
+                "BCP226",
+                "Expected at least one diagnostic at this location. Valid format is #disable-next-line diagnosticCode1 diagnosticCode2 ..."
```

Comment:
[nit] Would be good to put quotes (convention is double quotes) around the syntax example.

Reply:
Should also say: "Expected at least one diagnostic code..."

Reply:
Updated as part of this commit:
https://github.com/Azure/bicep/commit/7f012613ae1aad9fd2e8101ac82b73b362b8686b